scripts/prepare-for-release

```diff
@@ -130,11 +127,13 @@ update_versions() {
     for f in "${FILES[@]}"; do
         # do not exit if grep doesn't find $PREVIOUS_VERSION; continue to exit on all other exit codes
         has_incorrect_version=$(cat $f | grep "$PREVIOUS_VERSION" || [[ $? == 1 ]])
+        # escape the "." in version so that substitutions are done on EXACT version match only
+        previousVersionEsc="$(sed -e 's/\./\\./' <<< "$PREVIOUS_VERSION")"
```

Comment:
Does this sed work on both linux and mac?

Reply:
Good call-- I just tested on Linux and confirmed it's working